packages/arb-token-bridge-ui/src/components/syncers/ThemeIncluder.tsx

```diff
@@ -2,8 +2,8 @@ import dayjs from 'dayjs'
 import { useEffect, useState } from 'react'
 import { useTheme } from 'src/hooks/useTheme'
 
-const aprilFools = '2023-04-01'
-export const isAprilFools = dayjs().isSame(aprilFools, 'day')
+const aprilFools = '2023-04-01T08:00:00Z' // 8am UTC
+export const isAprilFools = dayjs().isAfter(aprilFools)
```

Comment:
Do you want to add the code that turns this off after 24 hours?

Reply:
I think we have that in place

Reply:
Where at? This logic just checks that it's after April 1st, but does not disable the boolean on April 2nd.

Reply:
Ah I see below